strecklista/templates/strecklista/components/quotecard.html

```diff
@@ -4,4 +4,7 @@
     <div class="quotee">{{ quote.who }}</div>
     <div class="detail">{{ quote.timestamp }}</div>
   </div>
+  <div class="quote-link">
+    Permanent länk: <a href="{{ request.get_host }}/quote/{{ quote.id }}">{{ request.get_host }}/quote/{{ quote.id }}</a>
```

Comment:
Anchor must have the protocol in `href`, otherwise `request.get_host` is interpreted as the protocol. Should be `href="//{{ request.get_host C}}[...]`, note the `//`. Isn't [{% url 'view.name' quote=quote.id %}](https://docs.djangoproject.com/en/2.0/ref/templates/builtins/#url) the correct way to do this though?

Also, `<a href="...">Permanent länk</a>` is IMHO better UX. It's how hyperlinks are intended to be used.